src/features/Dashboard/DashboardPage/index.scss

```
@@ -23,3 +30,20 @@
 .image-dashboard img {
   width: 100%;
 }
+
+@media (max-width: 768px) {
+  table,
+  th,
+  td {
+    font-size: 12px;
+  }
+
+  th {
+    text-align: center;
+  }
+
+  table {
+    display: block;
+    overflow-x: auto;
+  }
```

```suggestion
  }
  
  .license-section{
    margin-bottom: 1rem;
  }

  .instructor-assign-section {
    min-height: 200px;
  }
```
Add these styles in the media sections